views/merchant.erb

```diff
@@ -167,62 +170,104 @@
 
 <% if @client_token %>
   <script>
-    var client = braintree.setup('<%= @client_token %>', 'custom', {
-      id: "paymentForm",
-      hostedFields: {
-        onFieldEvent: function (event) {
-          if(event.type === "fieldStateChange" && !event.isValid){
-            console.log("No go! Clear loading state");
-            app.clearPaymentLoading();
-          }
-        },
-        styles: {
-          "input": {
-            "font-family": '"Open Sans Regular", "Helvetica Neue", "Helvetica", "Arial", Sans-serif',
-            "font-size": "16px",
-            "font-weight": "normal",
-            "color": "#3E3C41"
-          },
-          ":focus": {
-            "color": "#2CA1E9"
-          },
-        },
-        number: {
-          selector: "#card-number"
-        },
-        expirationDate: {
-          selector: "#expiration-date"
+    ;(function () {
```

Comment:
Neither are strictly required, but IMO nice to have, just wanted avoid declaring the form submit handlers as globals here since they're only used for the braintree web initialization